src/components/Modal.tsx

```diff
@@ -36,6 +36,14 @@ const Modal = ({ isOpen, onClose, children }: ModalProps) => {
                 leaveTo="opacity-0 scale-95"
               >
                 <Dialog.Panel className="transform overflow-hidden rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all">
+                  <div className="w-full -mb-6 relative flex justify-end">
+                    <button
+                      className=" text-gray-300 cursor-pointer hover:text-gray-700 focus:outline-none"
```

Comment:
please remove the blank space at the start of this string

Reply:
also `cursor-pointer` here is redundant as the `<button>` tag already makes the cursor a pointer